delatore/sources/awx_api.py

```diff
@@ -98,7 +99,11 @@ async def get_templates(self, filters: dict = None) -> Optional[list]:
                 response_data = await response.json()
             assert response.status == 200, f'Expected response 200, got {response.status}'
         try:
-            return response_data['results']
+            container = []
+            for i in range(len(response_data['results'])):
```

Comment:
You  can flatten list using [`itertools.chain.from_iterable`](https://docs.python.org/3/library/itertools.html#itertools.chain.from_iterable) or using nested list comprehension: https://stackoverflow.com/questions/952914/how-to-make-a-flat-list-out-of-list-of-lists